crypto/bio/bss_dgram.c

```diff
@@ -1918,20 +1900,19 @@ static void get_current_time(struct timeval *t)
 {
 # if defined(_WIN32)
     SYSTEMTIME st;
-    union {
-        unsigned __int64 ul;
-        FILETIME ft;
-    } now;
+    unsigned __int64 now_ul;
+    FILETIME now_ft;
 
     GetSystemTime(&st);
-    SystemTimeToFileTime(&st, &now.ft);
+    SystemTimeToFileTime(&st, &now_ft);
+    memcpy(&now_ul, &now_ft, sizeof(unsigned __int64));
```

Comment:
Wouldn't:
```c
now_ul = ((unsigned __int64)not_ft.dwHighDateTime << 32) + not_ft.dwLowDateTime;
```
be safer?  I know it's ever so slightly different to what the existing code did but it's endian friendly.

Reply:
Fixed. Thanks!